src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/AutoCrafterListener.java

```diff
@@ -38,6 +45,11 @@ public AutoCrafterListener(SlimefunPlugin plugin) {
     @EventHandler
     public void onInteract(PlayerRightClickEvent e) {
         Optional<Block> clickedBlock = e.getClickedBlock();
+        List<Recipe> recipes = org.bukkit.Bukkit.getRecipesFor(e.getItem());
+        Player p = e.getPlayer();
+        World w = p.getWorld();
```

Comment:
Variable not really needed. Also, please name this world

Reply:
Should I just not use those variables and directly put them in the code then? I just used variables because it feels more organized and the code doesn't get horizontally too long...